### PR TITLE
Bug 1984608: Set kube-scheduler leader election defaults

### PR DESCRIPTION
**Pull request by damemi**
This sets default LeaderElection values to better handle API server disruption as recommended in https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#handling-kube-apiserver-disruption

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/cc @soltysh

**Comment by damemi** [comment]
/hold
Something weird about how I tried to break out the common config in the first commit broke this in all the clusters I tried, but copying it to each profile seems to have fixed it. Want to examine more before merging but it seems like the 2nd commit works at least

**Comment by damemi** [comment]
/hold cancel
tests are green now, this is ready to tag

**Review by ravisantoshgudimetla — APPROVED** [review]
/lgtm

Thank you for working on this @damemi

**Pull request merged**
